src/main/http.ts

```diff
@@ -29,6 +37,85 @@ export class HttpServer extends Koa {
     @config({ default: 10000 }) HTTP_SHUTDOWN_DELAY!: number;
     @config({ default: 300000 }) HTTP_TIMEOUT!: number;
 
+    protected middlewares: MiddlewareSpec[] = [
```

Comment:
Since we extracted this as a separate field now, we could take it one step further and extract it to a separate class (or maybe a constant?) and then `@inject` it here. This way you don't have to redefine `CustomServer` every time you need to touch the middlewares.

@nicck WDYT?

Reply:
Yup, fair enough 🚀